src/logging.lua

```diff
@@ -29,7 +29,17 @@ local function LOG_MSG(self, level, fmt, ...)
   local f_type = type(fmt)
   if f_type == 'string' then
     if select('#', ...) > 0 then
-      local status, msg = pcall(format, fmt, ...)
+      local function error_handler(err)
+        local r = ''
+        local m = debug.traceback(err, 5)
+        for s in m:gmatch("(.-)\n") do
+          if s:match("%:%d+%:") then
+            r = r .. ' | ' .. s:gsub('\t', '')
+          end
+        end
+        return err .. r
+      end
```

Comment:
I think this creates a new closure on each call. It would be more efficient if the function `error_handler` was moved out of `LOG_MSG`.